common/src/main/java/tds/common/web/advice/ExceptionAdvice.java

```diff
@@ -36,6 +38,8 @@
     @ResponseStatus(value = HttpStatus.BAD_REQUEST)
     @ResponseBody
     ResponseEntity<ExceptionMessageResource> handleArgumentMismatchException(final MethodArgumentTypeMismatchException ex) {
+        LOG.info("Request made with invalid arguments ", ex);
```

Comment:
The reason I put it in as info is because this is an error based on the caller's requests not the microservice itself.  As in if a client was calling this I wouldn't want to be alerted that requests were failing due to this reason.  I would hope the client would log that because this error is a bad request.